runtime/CSharp/src/Dfa/AcceptStateInfo.cs

```diff
@@ -1,71 +0,0 @@
-/* Copyright (c) 2012-2017 The ANTLR Project. All rights reserved.
```

Comment:
Makes me nervous to delete files.

Reply:
Also, no mention in other runtimes and no doc.

Reply:
It's in the repo, but not even part of the project
@parrt Remember this was inherited from a diverging implementation